test/cartesian/Area.spec.tsx

```diff
@@ -329,3 +338,361 @@ describe.each(chartsThatDoNotSupportArea)('<Area /> as a child of $testName', ({
     expect(container.querySelectorAll('.recharts-curve')).toHaveLength(0);
   });
 });
+
+describe('getBaseValue', () => {
+  describe('when defined explicitly in props', () => {
+    it('should return number if baseValue is a number', () => {
+      // @ts-expect-error incomplete mock
+      const props: Props = {};
+      const item: Area = {
+        // @ts-expect-error incomplete mock
+        props: { baseValue: 8 },
+      };
+      // @ts-expect-error incomplete mock
+      const xAxis: Props['xAxis'] = {};
+      // @ts-expect-error incomplete mock
+      const yAxis: Props['yAxis'] = {};
+      const actual = Area.getBaseValue(props, item, xAxis, yAxis);
+      expect(actual).toBe(8);
+    });
+
+    it('should read baseValue from chart props, if item.props.baseValue is undefined', () => {
+      // @ts-expect-error incomplete mock
+      const props: Props = {
+        baseValue: 9,
+      };
+      const item: Area = {
+        // @ts-expect-error incomplete mock
+        props: { baseValue: undefined },
+      };
+      // @ts-expect-error incomplete mock
+      const xAxis: Props['xAxis'] = {};
+      // @ts-expect-error incomplete mock
+      const yAxis: Props['yAxis'] = {};
+      const actual = Area.getBaseValue(props, item, xAxis, yAxis);
+      expect(actual).toBe(9);
+    });
+
+    it('should prefer baseValue from Area props, if both are provided', () => {
+      // @ts-expect-error incomplete mock
+      const props: Props = {
+        baseValue: 9,
+      };
+      const item: Area = {
+        // @ts-expect-error incomplete mock
+        props: { baseValue: 10 },
+      };
+      // @ts-expect-error incomplete mock
+      const xAxis: Props['xAxis'] = {};
+      // @ts-expect-error incomplete mock
+      const yAxis: Props['yAxis'] = {};
+      const actual = Area.getBaseValue(props, item, xAxis, yAxis);
+      expect(actual).toBe(10);
+    });
+
+    it('should return number from domain when baseValue is NaN', () => {
+      // @ts-expect-error incomplete mock
+      const props: Props = {
+        layout: 'horizontal',
+      };
+      const item: Area = {
+        // @ts-expect-error incomplete mock
+        props: { baseValue: NaN },
+      };
+      // @ts-expect-error incomplete mock
+      const xAxis: Props['xAxis'] = {};
+      const yAxis: Props['yAxis'] = {
+        scale: {
+          // @ts-expect-error incomplete mock
+          domain: () => [30, 40],
+        },
+      };
+      const actual = Area.getBaseValue(props, item, xAxis, yAxis);
+      expect(actual).toBe(30);
+    });
+  });
+
+  describe.each(emptyBaseValues)('when baseValue is %s', baseValue => {
+    const testCases: ReadonlyArray<BaseValueTestCase> = [
+      { domain: [4, 5], expected: 4, axisType: 'number' },
```

Comment:
isn't `baseValue` supposed to be defaulted to 0?

Reply:
It defaults to the lower value of axis domain. Which defaults to 0. So yes, indirectly.